### ISSUE TITLE
Sidebar for a certain namespace

### ISSUE THREAD

**Issue by punk-t** [wwu-housing/starterbootstrap#68]
After hours of fiddling around and reading your issue #11 again and again I found out that the sidebar page has to be created in the root namespace of your wiki.
In contradiction to other templates (like dokuwiki default et al.) this root sidebar page is the only one to be interpreted. Sidebar pages inbetween any namespace are ignored in this template. Is this on purpose?

**Comment by apexskier** [comment]
This is not intentional, I'd suspect it results from the [page_findnearest](https://github.com/wwu-housing/starterbootstrap/blob/master/main.php#L13) function.

**Comment by apexskier** [comment]
Yup, take a look at the [function definition](http://xref.dokuwiki.org/reference/dokuwiki/nav.html?inc/pageutils.php.source.html#l735). It takes a pagename, not a namespaced location. 

This _should_ make it possible to have namespace specific sidebars, but I'm not seeing that work on an install I just tested on. Could be a caching issue though.

I'm not able to work on this currently, but submit a pull request if you are able to make it work better.
